content/applications/finance/accounting/bank/feeds/enablebanking.rst

```diff
@@ -0,0 +1,41 @@
+===============================================
+Enable Banking as bank synchronization provider
+===============================================
+
+**Enable Banking** is a third party provider aggregating banking information from bank accounts all
+in one place.
+
+.. image:: enablebanking/enablebanking.png
+   :align: center
+   :alt:   Enable Banking logo
+
+**Enable Banking** offers non-intrusive connectivity to ASPSPs’ official APIs across Europe without
+storing data. Odoo synchronizes directly with banks to get access to all bank transactions and
+automatically import them into your database.
```

Comment:
- it's weird you have a second paragraph about _both_ enablebanking at Odoo. Move the part about enablebank to the first paragraph --> "... all in one place. It offers non-intrusive connectivity..."  (--> see like ponto's doc) 
- use `'` instead of curly `’`   --> `ASPSPs' official`